src/common/config_opts.h

```diff
@@ -106,6 +106,8 @@ OPTION(async_compressor_threads, OPT_INT, 2)
 OPTION(async_compressor_thread_timeout, OPT_INT, 5)
 OPTION(async_compressor_thread_suicide_timeout, OPT_INT, 30)
 
+OPTION(plugin_crypto_accelerator, OPT_STR, "")
```

Comment:
is there a reason this doesn't have a default value? why not try to use acceleration by default?